Resources/Private/Partials/Registration/Fields/Datetime.html

```diff
@@ -0,0 +1,22 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
+      xmlns:e="http://typo3.org/ns/DERHANSEN/SfEventMgt/ViewHelpers"
+      data-namespace-typo3-fluid="true">
+
+<f:switch expression="{field.datepickermode}">
```

Comment:
I would add another getter `getDatepickermodeType()` to the `Field` model which contains the switch/case and returns the expected string. This way, a Unit test would be possible for the getter.

This would be better readable in Fluid template (e.g. `<f:form.textfield type="{field.datepickermodeType}" ...`